pkg/config/config/config.go

```diff
@@ -359,6 +361,14 @@ func (c SharingConfig) OptionsForContext(contextName string) SharingContext {
 		result.AutoAcceptTrusted = &falseVal
 	}
 
+	if result.AutoAcceptTrustedContacts == nil {
+		result.AutoAcceptTrustedContacts = &c.AutoAcceptTrustedContacts
+	}
+	if result.AutoAcceptTrustedContacts == nil {
+		falseVal := false
+		result.AutoAcceptTrustedContacts = &falseVal
+	}
```

Comment:
This block seems unnecessary since `&c.AutoAcceptTrustedContacts` cannot be `nil`.